nosnoc/nosnoc.py

```diff
@@ -1056,17 +1062,20 @@ def __init__(self, opts: NosnocOpts, model: NosnocModel, ocp: Optional[NosnocOcp
             print(f"{opts=}")
             print("\nerror creating solver for problem above:\n")
             print(f"\nerror is \n\n: {err}")
-            import pdb
-            pdb.set_trace()
+            breakpoint()
```

Comment:
do we really want to drop into PDB here?

Reply:
We also did before. I just change to breakpoint. We can also just raise the exception of course.

Reply:
I would raise an exception but that can be changed later.